Content.Server/Explosion/Components/OnUseTimerTriggerComponent.cs

```diff
@@ -44,5 +44,11 @@ public sealed class OnUseTimerTriggerComponent : Component
         /// </summary>
         [DataField("canToggleStartOnStick")]
         public bool AllowToggleStartOnStick;
+
+        /// <summary>
+        ///     Disallows starting the timer by hand, must be stuck or triggered by a system.
+        /// </summary>
+        [DataField("automated")]
+        public bool Automated;
```

Comment:
This timer is for on use, automated is the opposite so this should be its own component or an existing one if it exists.

Reply:
like AutomatedTriggerComponent then HasComp<AutomatedTriggerComponent> ?